src/app.js

```diff
@@ -24,8 +25,18 @@ const options = {
     showConsoleColors: true,
 };
 
+const idleTime = 1200000;
```

Comment:
In other js projects, we indicate constant values like this with ALLCAPS_SNAKE_CASE. I'd prefer to keep the convention in this repo too.